### PR TITLE
allow the use of enter key or space bar when a button is focused

### PR DESCRIPTION
**Pull request by kaligrafy**
for better accessibility. Note that the enter key is still disabled if the button is not focused with the tab key

### CODE REVIEW

**Comment by kaligrafy** [comment]
associated with https://github.com/chairemobilite/transition/pull/703

File: packages/evolution-legacy/src/components/survey/Button.js [hunk 1]
```diff
@@ -90,11 +90,12 @@ export class Button extends React.Component {
     return (
       <div className={widgetConfig.align || 'center'}>
         <button
-          type      = "button"
+          type = "button"
           className = {`survey-section__button button ${buttonColor} ${widgetConfig.size || 'large'} ${isLoading ? 'disabled' : ''}`}
           onMouseDown = {this.onMouseDown}
           onMouseUp = {this.onMouseUp}
-          disabled={isLoading}
+          onKeyUp = {(e) => { if (e.key === 'enter' || e.key === 'space' || e.which === 13 || e.which === 32) { this.onClickButton(); } else { return; } }}
```
Comment (tahini) [inline packages/evolution-legacy/src/components/survey/Button.js:97]: `yarn format` please

**Review by tahini — CHANGES_REQUESTED** [review]
Run `yarn format` on the code

**Comment by kaligrafy** [comment]
formatted

File: packages/evolution-legacy/src/components/survey/Button.js (hunk 1 above)
Comment (tahini) [reply packages/evolution-legacy/src/components/survey/Button.js:97]: haha, c'est même pas la ligne que j'attendais qui a été formatée!

**Review by tahini — APPROVED** [review]

**Pull request merged**